druntime/src/core/interpolation.d

```diff
@@ -0,0 +1,156 @@
+/++
+    This module provides definitions to support D's
+    interpolated expression sequence literal, sometimes
+    called string interpolation.
+
+
+    ---
+    string str;
+    int num;
+    // the compiler uses this module to implement the
+    // i"..." literal used here.
+    auto a = i"$(str) has $(num) items.";
```

Comment:
ddoc can't handle this line because ddoc tries to apply macros to D code. unbelievable.